Sources/TensorFlow/Core/Tensor.swift

```diff
@@ -158,7 +158,6 @@ public extension Tensor {
     @inlinable
     var array: ShapedArray<Scalar> {
         debugLog("Returning a host copy of array.")
-        internalConsistencyCheck(handle.isConcrete)
 	    return handle.makeHostCopy()
```

Comment:
```suggestion
      return handle.makeHostCopy()
```

Reply:
Thanks, it should be fixed now. I believe there was a mixture of some tabs in the file which caused the indentation issues.

Reply:
Thanks @jaingaurav! Yes it looked like some tabs were being used.